newIDE/app/src/stories/componentStories/BuildProgressAndActions.stories.js

```diff
@@ -23,55 +23,73 @@ export default {
 };
 
 export const Errored = () => (
-  <BuildProgressAndActions build={erroredCordovaBuild} game={game1} />
+  <BuildProgressAndActions
+    build={erroredCordovaBuild}
+    game={game1}
+    getThumbnailURL={buildId => undefined}
```

Comment:
There should be at least one instance of the component with the thumbnail for it to be tested in the storybook.